commands/help.rb

```diff
@@ -127,21 +129,29 @@ def self.help(command)
     when 'upgrade'
       puts <<~EOT
         Update package(s).
-        Usage: crew upgrade [-v|--verbose] [-s|--source] <package1> [<package2> ...]
+        Usage: crew upgrade [-f|--force] [-s|--source] [-v|--verbose] <package1> [<package2> ...]
         If package(s) are omitted, all packages will be updated.  Otherwise, specific package(s) will be updated.
         Use `crew update` to update crew itself.
+        If `-f` or `--force` is present, interactive prompts will be disabled.
         If `-s` or `--source` is present, the package(s) will be compiled instead of upgraded via binary.
         If `-v` or `--verbose` is present, extra information will be displayed.
       EOT
     when 'upload'
       puts <<~EOT
         Upload binaries.
-        Usage: crew upload [<package1> <package2> ...]
+        Usage: crew upload [-f|--force] [-v|--verbose] [<package1> <package2> ...]
         This will update the binary_sha256 hashes in each package and upload binaries to GitLab.
         The GITLAB_TOKEN environment variable must be set to access the upstream repository.
         If no package(s) are provided, all binaries in `#{CREW_LOCAL_REPO_ROOT}/release/<arch>` will be uploaded.
+        If `-f` or `--force` is present, interactive prompts will be disabled.
```

Comment:
Does upload actually have any behavior for force?

Reply:
Good observation.  Added logic.